virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.adoc

```diff
@@ -4,17 +4,7 @@ include::modules/virt-document-attributes.adoc[]
 :context: virt-viewing-ip-of-vm-vnic
 toc::[]
 
-The QEMU guest agent runs on the virtual machine and passes the IP address of attached NICs to the host, allowing you to view the IP address from both the web console and the `oc` client.
-
-== Prerequisites
-
-. Verify that the guest agent is installed and running by entering the following command:
-+
-----
-$ systemctl status qemu-guest-agent
-----
-+
-. If the guest agent is not installed and running, xref:../../../virt/virtual_machines/vm_networking/virt-installing-qemu-guest-agent.adoc#virt-installing-qemu-guest-agent[install and run the guest agent on the virtual machine].
+You can view the IP address for a Network Interface Card (NIC) by using the web console or the `oc` client. If the xref:../../../virt/virtual_machines/vm_networking/virt-viewing-qemu-guest-agent-web.adoc#virt-viewing-qemu-guest-agent-web[QEMU guest agent] runs on the virtual machine, you can view additional information about the virtual machine, users, and file systems.  
```

Comment:
And information about secondary networks

Reply:
@phoracek - Please note. I am removing the second sentence because this entire module is focused only on viewing the IP address of the NIC and this content resides in the VM Networking section. A mention about viewing logged in users, file systems etc does not belong to this content module - all of that information will now be available in the Virtual Machines section.

All information about the QEMU guest agent (installing and viewing) is being moved to the Virtual Machines section.

Reply:
Understood. Please let me know once it is moved and ready for another review

Reply:
Content was moved out to new location.